systemtest/src/main/java/io/strimzi/systemtest/logs/LogCollector.java

```diff
@@ -305,6 +315,24 @@ private void collectClusterInfo(String namespace) {
         writeFile(this.testSuite + "/cluster-status.log", nodes);
     }
 
+    private void collectOperatorGroups(String namespace) {
+        LOGGER.info("Collecting operatorGroups");
+        String nodes = cmdKubeClient(namespace).exec(false, Level.DEBUG, "describe", "operatorGroups").out();
+        writeFile(namespaceFile + "/operator-groups.log", nodes);
+    }
+
+    private void collectSubscriptions(String namespace) {
+        LOGGER.info("Collecting subscriptions");
+        String nodes = cmdKubeClient(namespace).exec(false, Level.DEBUG, "describe", "subscriptions").out();
+        writeFile(namespaceFile + "/subscriptions.log", nodes);
+    }
+
+    private void collectClusterServiceVersions(String namespace) {
+        LOGGER.info("Collecting clusterServiceVersions");
+        String nodes = cmdKubeClient(namespace).exec(false, Level.DEBUG, "describe", "clusterServiceVersions").out();
+        writeFile(namespaceFile + "/cluster-service-versions.log", nodes);
+    }
```

Comment:
Would `-o yaml` instead of `describe` be better? At least in terms of readability?